tests/visuals/test_plots.py

```diff
@@ -213,7 +236,7 @@ def test_traces(self, distributions, traces):
         )
 
         shown_traces = pd.DataFrame(vega.find_named(schema["data"], "traces")["values"])
-        plots.save_schema(schema, "_schema.json")
+        plots.save_schema(schema, "test_traces_schema.vg.json")
```

Comment:
Please change back to `_schema.json`.  This keeps us from creating a lot of temporary files (and thus keeping things clean). If we want to go to individually-named files, we should use a directory that is ignored by git.